app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -5,12 +5,40 @@
       "integration": "Integration",
       "integrations": "Integrations"
     },
+    "connections": {
+      "connection": "Connection",
+      "connections": "Connections"
+    },
     "dashboard": {
-      "header": "Dashboard",
+      "heading": "Dashboard",
       "metrics": {
         "sysmetrics": "System Metrics",
-        "uptime": "Uptime",
-        "integrationsheader": "{{0}} {{integrations.integrations}}"
+        "integrationsheader": "{{0}} {{integrations.integrations}}",
+        "totalmessages": "Total Messages",
+        "uptime": {
+          "uptime": "Uptime",
+          "since": "Since {{0}}"
+        },
+        "create": "{{create}} {{integrations.integration}}",
+        "viewall": "View All {{integrations.integrations}}"
+      },
+      "connections": {
+        "heading": "{{connections.connections}}",
+        "viewall": "View All {{connections.connections}}",
```

Comment:
This could be removed along with `metrics.viewall` and replaced in the template by a generic shared `viewall` key populated as `view All {{0}}` so we can inject whatever value we need